components/ntp_background_images/browser/ntp_background_images_service.cc

```diff
@@ -204,7 +204,7 @@ void NTPBackgroundImagesService::RegisterSponsoredImagesComponent() {
   }
   sponsored_images_component_id_ = data->component_id.data();
 
-  DVLOG(2) << __func__ << ": Start NTP SI component";
+  VLOG(6) << "Register NTP Sposored Images component";
```

Comment:
We already log the same information in `RegisterBackgroundImagesComponent()

Reply:
In the `RegisterBackgroundImagesComponent` we logged:
VLOG(6) << "Register NTP Background Images component";

Reply:
spelling error: `Sposored`

Reply:
Fixed